README.md

```diff
@@ -45,10 +45,32 @@ Mixins within `o-brand` help configure components to support brands. There is no
 
 The following mixins and functions help brand a component.
 
+- [oBrandGetCurrentBrand](#obrandgetcurrentbrand) - Set a default brand if necessary
 - [oBrandDefine](#obranddefine) - Define brand configuration (variables & supported variants).
 - [oBrandGet](#obrandget) - Retrieve brand variables.
 - [oBrandSupportsVariant](#obrandsupportsvariant) - Check if the brand supports a variant.
 
+### oBrandGetCurrentBrand
+
+Certain components only require one brand to be defined, and all other components only need to define one brand at a time.
+
+In order to allow conditional configuration, this function is used to carry forward a parent brand, or set a default brand in case none has been defined.
+If a brand that is not supported by `o-brand` has been set, this function will throw an error. It will run in `oBrandDefine` and `oBrandGet` as a safe guard.
+
+If it is being used for conditional brand setting, as below, it should be set to the `$o-brand` variable, and should be the first function in a components brand configuration file.
+
+```scss
+$o-brand: oBrandGetCurrentBrand();
```

Comment:
The user sets `$o-brand` on a project level to configure their brand. Might want to use a different variable name or just call the function in the conditional:

```
@if oBrandGetCurrentBrand() == 'master' {
	oBrandDefine('master' ...)
}
```

Reply:
Maybe this doesn't need to be a new section in the readme but oBrandDefine examples should be updated instead?

Reply:
I think it needs a little bit of explanation should we want to use it for the conditional cases 🤔